README.md

```diff
@@ -986,16 +988,22 @@ Latest stable release, with native support for the Kafka 0.10 protocol and event
 
 ## Higher level libraries
 
-Currently, there are three actively developed frameworks based on ruby-kafka, that provide higher level API that can be used to work with Kafka messages:
+Currently, there are three actively developed frameworks based on ruby-kafka, that provide higher level API that can be used to work with Kafka messages and two libraries for publishing messages.
 
-* [Racecar](https://github.com/zendesk/racecar) - A simple framework that integrates with Ruby on Rails to provide a seamless way to write, test, configure, and run Kafka consumers. It comes with sensible defaults and conventions.
+### Frameworks
```

Comment:
This might better be described as _message processing frameworks_.

Reply:
not quite sure about that though as karafka is also a message producing framework and there's an issue for that in racecar as well

Reply:
I think _processing_ typically includes producing messages, as well – for instance, Kafka Streams is described as _stream processing_.

Reply:
@dasch  good point. Will update accordingly.